.github/workflows/tox-test.yml

```diff
@@ -24,14 +24,26 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
 
-      - name: Install Python Dependencies
-        uses: HassanAbouelela/actions/setup-python@setup-python_v1.4.0
-        id: setup_python
+      - name: Setup poetry
+        id: poetry_setup
+        uses: ItsDrike/setup-poetry@v1
         with:
-          install_args: "--with workflow-tox --without lint"
-          python_version: "${{ matrix.python-version }}"
+          install-args: "--with workflow-tox --without lint"
+          python-version: "${{ matrix.python-version }}"
+
+      - name: Pytest Cache Caching
+        uses: actions/cache@v3
+        with:
+          path: ${{ github.workspace }}/.pytest_cache
+          key: "pytest-${{ runner.os }}-${{ steps.poetry_setup.outputs.python-version }}-\
+                ${{ hashFiles('./tests/**', './poetry.lock', './pyproject.toml') }}"
```

Comment:
BTW, what about providing `hash-key` output in `setup-poetry` action? It will include all necessary information to add in cache key, like Python version, OS, `poetry.lock` and `pyproject.toml` hash, etc. I can open an issue in that action, if you prefer so.

Reply:
I don't think this is particularly necessary, especially as it would mean I'd probably have to add 2 more outputs, for both the cache key of the poetry installation itself, and for cache key the virtual environment with dependencies.

Either way, if you want to discuss this further, feel free to open an issue in [the action's repo](https://github.com/ItsDrike/setup-poetry/issues).